queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-01-01';
```

Comment:
- [ ] It would be nice if you include the whole year 2019 in the query since you are supposed to get the animals from 2016 to 2019 so it would be better if you say `🟢 2019-12-31` instead of `🔴 2019-01-01` 👍 to follow the project requirements 😊